README.md

```diff
@@ -2,6 +2,71 @@
 
 `image-tools` is a collection of tools for working with the [OCI image format specification](https://github.com/opencontainers/image-spec).
 
+# Install
+
+it is recommended that use `go` to download a single command tools.
+
+```
+$ go get -d github.com/opencontainers/image-tools/cmd/oci-unpack
+$ cd $GOPATH/src/github.com/opencontainers/image-tools/
+$ make all
+$ sudo make install
+```
+
+## uninstall
+
+```
+$ sudo make uninstall
+```
+
+## example
+
+### Obtaining an image
+
+The following examples assume you have a (image-layout)[https://github.com/opencontainers/image-spec/blob/v1.0.0-rc2/image-layout.md] tar archive at `busybox-oci`.
```

Comment:
The syntax of URL link seems incorrect. Should be [abbreviation](url)
And please check other positions in this commit.